apps/web/src/ApplicationReadyGuard.tsx

```diff
@@ -34,7 +34,7 @@ export function ApplicationReadyGuard({ children }: PropsWithChildren<{}>) {
   function isOnboardingComplete() {
     if (IS_EE_AUTH_ENABLED) {
       // TODO: replace with actual check property (e.g. isOnboardingCompleted)
-      return currentOrganization?.productUseCases !== undefined;
+      return currentOrganization?.productUseCases !== undefined || currentOrganization?.language !== undefined;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏ I'm looking at this and thinking is this actually an organization level question? Should this maybe be asked on a per user basis?

Reply:
It looks to me like an org owner question, its filled once when org is created. Imo the goal is to collect languages/technologies that are used within the organization, not sure if it would help us to collect it per user basis.

Reply:
fair enough